src/components/shared/ChatMessageItem.tsx

```
@@ -468,48 +560,61 @@ function ChatMessageItem({
-                                  ? 'text-[hsl(var(--foreground)_/_0.8)] dark:text-purple-300'
-                                  : 'text-[hsl(var(--foreground)_/_0.8)] dark:text-[hsl(var(--muted-foreground))]',
-                              )}
-                            >
-                              <span>{formattedTimestamp}</span>
-                              {isSender && (
-                                <CheckCheck className="w-3.5 h-3.5" />
+                                'w-full break-all overflow-wrap-anywhere',
+                                'text-4xl leading-snug text-center break-normal',
                               )}
+                              style={{ wordBreak: 'break-word', overflowWrap: 'anywhere' }}
+                              dangerouslySetInnerHTML={{
```

Biome (2.3.13)

[error] Avoid passing content using the dangerouslySetInnerHTML prop.

Setting content using code can expose users to cross-site scripting (XSS) attacks

(lint/security/noDangerouslySetInnerHtml)